cmd/exporters/prometheus/httpd.go

```diff
@@ -136,6 +147,32 @@ func (me *Prometheus) ServeMetrics(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func filterMetaTags(metrics [][]byte) [][]byte {
```

Comment:
Could you add a comment or unit test with an example of a line that will be filtered because of meta tag check?

Reply:
done

> Thanks @vgratian - agreed that it would be nice to have the validator in Go instead of Python, but not required right now.
> 
> Maybe we can use another projects's open-metrics endpoint validation in the future as another validator.
> 
> Minor, but might be worth squashing your 30 commits into one before be rebase and merge.

I tried to squash the commits, but I fear I messed up somehow. Should I maybe squash the commits into a new branch and re-open a PR?

Reply:
Yes please try that. You additional comment looks good

Reply:
> Yes please try that. You additional comment looks good

@cgrinds, opened a new PR: #211

Reply:
please ignore last comment, we are able to squash from Github when merging